app/models/grda_warehouse/hud/client.rb

```diff
@@ -815,6 +819,7 @@ def self.extended_age_groups
     end
 
     def source_clients_searchable_to(user)
+      # FIXME: looks like we intended to memoize, it's not working due to missing conditional assignment
```

Comment:
I think that's a correct assessment.  This should be `||=`  I'm not seeing an obvious reason it couldn't be memoized that way.